src/web3-utils.js

```diff
@@ -8,21 +8,49 @@ import Web3 from 'web3'
 const ANY_ADDRESS = '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF'
 const EMPTY_ADDRESS = '0x0000000000000000000000000000000000000000'
 
-// Check address equality without checksums
+/** 
+ * Check address equality without checksums
+ * @param {string} first - First address
+ * @param {string} second - Second address
+ * @returns {boolean} - Address equality
+ */ 
 export function addressesEqual(first, second) {
   first = first && first.toLowerCase()
   second = second && second.toLowerCase()
   return first === second
 }
 
-// Shorten an Ethereum address. `charsLength` allows to change the number of
-// characters on both sides of the ellipsis.
-//
-// Examples:
-//   shortenAddress('0x19731977931271')    // 0x1973…1271
-//   shortenAddress('0x19731977931271', 2) // 0x19…71
-//   shortenAddress('0x197319')            // 0x197319 (already short enough)
-//
+/**
+ *  Takes a Wei value and returns it converted to the specified Ether subunit rounded to the specified number of digits after the decimal point.
+ * @param {string} wei - Value in Wei
+ * @param {number} [digits=2] - Number of digits after the decimal point (by default 2)
+ * @param {string} [unit='ether'] - Unit to convert to (by default 'ether')
+ * @return {string} - The converted Wei value to the specified Ether subunit and rounded to the specified digits after the decimal point
+ */
+export function fromWeiRounded(wei, digits = 2, unit = 'ether') {
+  let unitValue = Web3.utils.fromWei(wei, unit)
+  const decimalIndex = unitValue.indexOf('.')
+  if(decimalIndex != -1) {
+    unitValue = unitValue.substring(0, decimalIndex + digits + 2)
+    const roundedLastDigit = unitValue[unitValue.length-2].concat('.' + unitValue[unitValue.length -1])
+    unitValue = unitValue.substring(0, unitValue.length - 2).concat(Math.round(roundedLastDigit))
```

Comment:
I think this is a little bit overkill for what we're trying to achieve.

We could simply get the last digit that we care about, and do a `lastDigit >= 5 ? lastDigit + 1 : lastDigit` calculation instead.